### ISSUE TITLE
Error parsing SELECT statement combining literal with aggregate functions (missing ENDSELECT)

### ISSUE THREAD

**Issue by PilotFlying** [SAP/abap-cleaner#165]
Both of these
```
    SELECT 'MATKL'    AS someliteral,
           COUNT( * ) AS somecount
      FROM mara
     WHERE matnr = '123'
      INTO @DATA(ls).
      
    SELECT FROM mara
      FIELDS 'MATKL'    AS someliteral,
             COUNT( * ) AS somecount
     WHERE matnr = '123'
      INTO @DATA(ls2).
```
result in
```
expected ENDSELECT, but found ENDMETHOD. Opening command (line 212): SELECT FROM mara [...]
```

(Thanks!)

**Comment by jmgrassau** [comment]
Hi Constantinos, 

thanks a lot for reporting this! This bug will be fixed in the next release. 

Kind regards,
Jörg-Michael

**Comment by PilotFlying** [comment]
Hi Jörg-Michael, thanks - and also for this great tool!

**Comment by jmgrassau** [comment]
Hi Constantinos, 

thanks again – this should now be fixed in version 1.8.0, which was just released!

Kind regards,
Jörg-Michael
